contracts/sc_erc721/ERC721ServiceChain.sol

```diff
@@ -42,7 +42,6 @@ contract ERC721ServiceChain is ERC721, Ownable {
 
     function requestValueTransfer(uint256 _uid, address _to, bytes calldata _extraData) external {
         transferFrom(msg.sender, bridge, _uid);
-
```

Comment:
Do we really need this change?

Reply:
Is not it an unnecessary newline?

Reply:
The only change in this file is removing the newline.
In this case, I suggest you to make another PR to avoid any confusion of readers.

Reply:
Reverted